go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/ipfs/dir-index-html v1.0.3
-	github.com/ipfs/go-bitswap v0.1.6
+	github.com/ipfs/go-bitswap v0.1.5
```

Comment:
Looks like these are bad merges. It should use `btfscoin` branch's latest versions.